tests/common/snappi/qos_fixtures.py

```diff
@@ -106,3 +106,72 @@ def lossy_prio_list(all_prio_list, lossless_prio_list):
     """
     result = [x for x in all_prio_list if x not in lossless_prio_list]
     return result
+
+
+def prio_dscp_map_dut_base(duthost):
```

Comment:
Why not call prio_dscp_map_dut_base in prio_dscp_map?

Reply:
this has been resolved, redundant functions have been removed